docs/resources/team.md

```diff
@@ -33,3 +33,28 @@ resource "buildkite_team" "team" {
 * `id`   - The GraphQL ID of the team.
 * `slug` - The name of the team.
 * `uuid` - The UUID for the team.
+
+## Import
+
+Teams can be imported using the `GraphQL ID` (not UUID), e.g.
+
+```
+$ terraform import buildkite_team.fleet UGlwZWxpbmUtLS00MzVjYWQ1OC1lODFkLTQ1YWYtODYzNy1iMWNmODA3MDIzOGQ=
+```
+
+To find the ID to use, you can use the GraphQL query below. Alternatively, you could use this [pre-saved query](https://buildkite.com/user/graphql/console/9cfcca03-df1a-4951-9fa9-a7345aa3daaf2), where you will need fo fill in the organization slug and search term (TEAM_SEARCH_TERM) for the team.
```

Comment:
I think this saved query URL isn't correct. When I navigate, it shows me this query:

```graphql
query pipelineId {
  pipeline(slug: "bk-testing") {
    id
  }
}
```